release-notes.md

```diff
@@ -354,6 +360,50 @@
     - [Unit testing](#unit-testing)
     - [Integration tests parallelization](#integration-tests-parallelization)
 
+# v44.0
+
+This release has been tracked in our [roadmap
+project](https://github.com/orgs/cloud-hypervisor/projects/6) as iteration
+v44.0. The following user visible changes have been made:
+
+### Configurable `virtio-iommu` Address Width
+
+The `iommu_address_width` option has been added to `--platform` to allow users
+to limit the `virtio-iommu` address space in the guest. (#6900)
+
+### Notable Performance Improvements
+
+The `VIRTIO_BLK_F_SEG_MAX` feature has been enabled for `virtio-block` devices,
+which brings significant performance improvements on throughput. (#6885)
+
+The `io_uring` entries are no longer forced to use async helper workers,
+delegating the decision to the kernel. This change resolved the issue of having
+excessive amount of worker threads when `io_uring` is being used, which is
+expected to improve performance, such as reducing memory usage and reduce CPU
+contention.
+
```

Comment:
nit: Extra whitespace line.

Reply:
Good catch. I am now landing this PR and going to make the release shortly.